crates/bevy_pbr/src/render/morph.wgsl

```diff
@@ -61,4 +65,11 @@ fn morph(vertex_index: u32, component_offset: u32, weight_index: u32) -> vec3<f3
     );
 }
 
+#ifdef MOTION_VECTOR_PREPASS
+fn previous_weight_at(weight_index: u32) -> f32 {
+    let i = weight_index;
+    return previous_morph_weights.weights[i / 4u][i % 4u];
```

Comment:
I hope the compiler would do this but...
```suggestion
    return previous_morph_weights.weights[i >> 2u][i & 3u];
```

Reply:
I know it's pretty minor, but could we leave this off to a different PR? Otherwise  I would have to update more code (the original `weight_at` in the same file uses `/` and `%`) I think it makes more to keep the two implementations in sync.

Reply:
LLVM will definitely do this in instcombine.